payments-core/src/main/java/com/stripe/android/payments/bankaccount/domain/RetrieveStripeIntent.kt

```diff
@@ -0,0 +1,24 @@
+package com.stripe.android.payments.bankaccount.domain
+
+import com.stripe.android.core.networking.ApiRequest
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.networking.StripeRepository
+import javax.inject.Inject
+
+internal class RetrieveStripeIntent @Inject constructor(
+    private val stripeRepository: StripeRepository
+) {
+
+    /**
+     * Retrieve [StripeIntent].
+     */
+    suspend fun retrieve(
```

Comment:
calling this retrieve is redundant, let's use the interactor pattern
```suggestion
    suspend operator fun invoke(
```